lib/librato/rails/configuration.rb

```diff
@@ -8,14 +8,22 @@ module Rails
     # https://github.com/librato/librato-rack/blob/master/lib/librato/rack/configuration.rb
     #
     class Configuration < Rack::Configuration
-      CONFIG_SETTABLE = %w{user token flush_interval log_level prefix source source_pids proxy}
+      CONFIG_SETTABLE = %w{user token flush_interval log_level prefix source source_pids proxy suites}
+
+      DEFAULT_SUITES = [:rails_cache, :rails_controller, :rails_render, :rails_sql, :rails_mail, :rails_job]
 
       attr_accessor :config_by, :config_file
 
       # options:
       # * :config_file - alternate config file location
       #
       def initialize(options={})
+        # Update the set of default suites (remove and re-set const to avoid
+        # warnings)
+        suites = Librato::Rack::Configuration::DEFAULT_SUITES | DEFAULT_SUITES
+        Librato::Rack::Configuration.send(:remove_const, :DEFAULT_SUITES)
+        Librato::Rack::Configuration.const_set(:DEFAULT_SUITES, suites)
```

Comment:
this feels a bit gross, can this be solved by overriding in the subclass? if not, maybe lib-rack should call a private method to retrieve the constant value so we can override it more cleanly here?